v2/pkg/protocols/common/interactsh/interactsh.go

```diff
@@ -49,7 +52,7 @@ type Client struct {
 
 var (
 	defaultInteractionDuration = 60 * time.Second
-	interactshURLMarker        = "{{interactsh-url}}"
+	interactshURLMarkerRegex   = regexp.MustCompile(`{{interactsh-url(\S){0,3}}}`)
```

Comment:
Instead of the above regex, we can make it more strict by using the below regex - 

```
{{interactsh-url(?:_[0-9]+)}}
```

Also, we have to decide whether we want to use `_` or `-` for separating the interactsh numbered placeholders.